cpp/benchmarks/fixture/nvbench_fixture.hpp

```diff
@@ -25,12 +26,18 @@
 #include <rmm/mr/device/owning_wrapper.hpp>
 #include <rmm/mr/device/per_device_resource.hpp>
 #include <rmm/mr/device/pool_memory_resource.hpp>
+#include <rmm/mr/host/host_memory_resource.hpp>
+#include <rmm/mr/host/new_delete_resource.hpp>
+#include <rmm/mr/host/pinned_memory_resource.hpp>
```

Comment:
This is the old, to be deprecated pinned memory resource. Please use `<rmm/mr/pinned_host_memory_resource.hpp>